CubeLineMoment.py

```diff
@@ -153,9 +160,26 @@ def cubelinemoment_setup(cube, cuberegion, cutoutcube,
         except AttributeError:
             cutoutcube = cutoutcube.subcube_from_regions(regions.Regions.read(cutoutcuberegion))
     noisecubebright = cutoutcube
-    # For the NGC4945 Band 6 data use the C18O 2-1 line in spw1 for the dense
-    # gas mask for all Band 6 lines.
-    #cutoutcube = SpectralCube.read('NGC4945-H213COJ32K1-Feather-line.fits').with_spectral_unit(u.Hz).subcube_from_regions(regions.read_ds9('ngc4945boxband6.reg'))
+
+    # MOVED SAMPLE_PIXEL INTERPRETATION HERE...
+    if sample_pixel != None:
+        # Check to make sure that sample pixexl regions file exists.  Open it if
+        #  it does exist, and exit script if it does not exist.
+        if os.path.isfile(sample_pixel):
+            try:
+                regsample = regions.read_ds9(sample_pixel)
+            except AttributeError:
+                regsample = regions.Regions.read(sample_pixel)
+        else:
+            raise ValueError("Sample pixel file {0} does not exist.".format(sample_pixel))
+
+    sample_pixel_list = []
+    regionlabel = []
+    for point in regsample:
+        sample_pixel_list.append((int(point.to_pixel(wcs.WCS(cube.header)).center.x), int(point.to_pixel(wcs.WCS(cube.header)).center.y),point.meta.get('text')))
```

Comment:
```suggestion
    ww = cube.wcs.celestial
    for point in regsample:
        xx, yy = map(int, point.to_pixel(ww))
        sample_pixel_list.append(xx, yy, point.meta.get('text')))
```

Reply:
Oops, my suggestion here was wrong.  This
```python
        xx, yy = map(int, point.to_pixel(ww))
```
needs to be 
```python
        cen = point.to_pixel(ww).center
        xx, yy = int(cen.x), int(cen.y)
```
or similar.  Better check that this works before we hit merge

Reply:
Tried this change on my local copy (does not include the other changes in this PR).  I think it should be:
```
    ww = cube.wcs.celestial
    for point in regsample:
        cen = point.to_pixel(ww).center
        xx, yy = int(cen.x), int(cen.y)
        sample_pixel_list.append((xx, yy, point.meta.get('text')))
```
Note the extra `(` in the `append` call.

Reply:
I am afraid that I don't know how to make this change to the code so that it gets included in the merge.

Reply:
Just edit it in to the PR the same way you edited the code up to this point.  Usual process on your computer:
```
git pull
<make changes>
git commit
git push
```
You don't need to worry about my part of this - I made a mistake anyway.

Reply:
Fixed.